_content/gateways/faq/index.md

```diff
@@ -58,18 +58,16 @@ The LED lights are a great way of understanding the current state of the gateway
 
 #### Q. What if I take the gateway to a different location. Do I have repeat the configuration?
 
-Once you configure the gateway, the settings are locked to that location. So, in case you move to a different location you will have to do the activation again. 
-
-Goto  (https://ttn.fyi/activate) and at Step 1, you can select your gateway from the list and configure them.
+No you don't have to re-configure. For better accuracy, it is recommeded to however update the location settings in the Console (Settings->Location).
```

Comment:
> recommeded

Typo + let's rephrase that question differently, e.g. "How do I change my gateway's location?"

Reply:
What steps do I need to take if I want to relocate my gateway?

There is no need to re-configure it in this case. However, for better accuracy, it is recommended to however update the location settings in the Console (Settings->Location).

Reply:
I think we can skip the reconfiguration sentence, for less confusion. For example:

> **If I change the location of my gateway, how can I change its location?**

> You can update the location settings in the Console. Go to your gateway's settings, and in the Location section, select the new location of your gateway.